.github/workflows/goreleaser-build-publish-develop.yml

```diff
@@ -0,0 +1,43 @@
+name: "Build publish Chainlink develop on private ECR"
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  push-chainlink-develop:
+    # runs-on: ubuntu-20.04
+    runs-on:
+      # labels: ubuntu20.04-8cores-32GB
+      labels: ubuntu20.04-16cores-64GB
+    environment: build-develop
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # v1.7.0
+        with:
+          role-to-assume: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN }}
+          role-duration-seconds: ${{ secrets.AWS_ROLE_DURATION_SECONDS }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Build, sign, and publish image
+        uses: ./.github/actions/goreleaser-build-sign-publish
+        with:
+          enable-docker-publish: "true"
+          docker-registry: ${{ secrets.AWS_DEVELOP_ECR_HOSTNAME }}
```

Comment:
Does this change the repo name from https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/build-publish-develop.yml#L27 ?

If so, we probably need to duplicate this step to publish to this one _and_ to the existing one until all other testing/tools are pointing to the new one.

Reply:
The repo name will come from the goreleaser config. 
image template: https://github.com/smartcontractkit/chainlink/blob/feat/re-205/.goreleaser.develop.yaml#L105
project name: https://github.com/smartcontractkit/chainlink/blob/feat/re-205/.goreleaser.develop.yaml#L2

Reply:
hmm, we cant duplicate this step unless we publish to a different repo. Tags will conflict...

Reply:
repo name should be the same, its just configured in a different location, resolving..

Reply:
I see. I think we want to publish to two repo's temporarily: `chainlink` and `chainlink-develop`.

Then we can close this https://github.com/smartcontractkit/chainlink/pull/7975 -- what's the best way to support that? If it's a pain, feel free to let me know, resolve, and I can handle it in a separate PR.

Reply:
implemented here: https://github.com/smartcontractkit/chainlink/pull/8073/commits/b5c65d0bb0fa2be38becaeaba567ac35b55c99e1